src/main/java/org/broadinstitute/dsm/model/ParticipantWrapper.java

```diff
@@ -375,10 +379,13 @@ private static void updateProbandDataIfESParticipantUpdated(DDPInstance instance
         }
     }
 
-    private static void extractAndInsertProbandFromESData(DDPInstance instance, Map<String, Object> esData) {
+    private static void extractAndInsertProbandFromESData(DDPInstance instance, Map<String, Object> esData,
```

Comment:
I wonder if we should move these methods which are used only for study with ADD_FAMILY_MEMBER into a separate study class